lib/guardian/db/adapter/ets.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Guardian.DB.ETSAdapter do
+  @moduledoc """
+  Implement the Guardian.DB.Adapter for ETS
+  """
+
+  @behaviour Guardian.DB.Adapter
+
+  @impl true
+  def one(claims, opts) do
+    jti = Map.get(claims, "jti")
+    aud = Map.get(claims, "aud")
+
+    match =
+      opts
+      |> Keyword.fetch!(:table)
+      |> :ets.match({jti, aud, :_, :_, :"$1"})
+
+    case match do
+      [[token]] -> token
+      _ -> nil
+    end
+  end
+
+  @impl true
+  def insert(%{valid?: true} = changeset, opts) do
+    table = Keyword.fetch!(opts, :table)
+    token = Map.merge(changeset.data, changeset.changes)
+
+    unless :ets.insert(table, {token.jti, token.aud, token.sub, token.exp, token}) do
+      raise """
+      An error occurred trying to insert a new record into the ETS table #{table}. 
+
+      Please ensure you've created the table before attempting to insert records.
+      """
+    end
+
+    {:ok, token}
+  end
+
+  def insert(changeset, _opts) do
+    {:error, changeset}
+  end
+
+  @impl true
+  def delete(%{jti: jti} = token, opts) do
+    table = Keyword.fetch!(opts, :table)
+
+    if :ets.delete(table, jti) do
+      {:ok, token}
+    else
+      {:error, token}
```

Comment:
What about adding an error message here? Even if we don't have the reason from `:ets`

Reply:
Any suggestions for this @aleDsz?

Reply:
Sorry for the delay, what about `{:error, "Failed to delete from ETS table"}`?